app/webpacker/packs/application.js

```diff
@@ -1,6 +1,9 @@
 /* eslint-disable import/first */
 require.context("govuk-frontend/govuk/assets");
 
+//= require govuk_publishing_components/dependencies
+//= require govuk_publishing_components/all_components
```

Comment:
What does this do?

Reply:
Got it from https://github.com/alphagov/govuk_publishing_components/blob/master/docs/install-and-use.md

I presume it adds any javascript that is needed to make govuk publishing components work.

That being said, maybe I should switch to a version that is not deprecated.

Reply:
Also, might be worth digging to see if there is any that we want, actually - maybe all we want is their css.

Reply:
So I had a quick read through their js, and I am tempted to just not use it. Until some extensive and deep testing shows we need it, we can probably cut it.